vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -616,6 +625,21 @@ void finish_multiline_example(vw& all, ccb& data, multi_ex& ec_seq)
   VW::finish_example(all, ec_seq);
 }
 
+void save_load(ccb& sm, io_buf& io, bool read, bool text)
+{
+  if (io.num_files() == 0) { return; }
+
+  // We want to enter this block if either we are writing, or reading a model file after the version in which this was
+  // added.
+  if (!read || sm.model_file_version >= VERSION_FILE_WITH_CCB_MULTI_SLOTS_SEEN_FLAG)
+  {
+    std::stringstream msg;
+    if (!read) { msg << "CCB: has_seen_multi_slot_example = " << sm.has_seen_multi_slot_example << "\n"; }
```

Comment:
why do we need this exactly?

Reply:
The feature or saving and loading from models?

Reply:
saving and loading, wouldn't version number be enough? o why do we need to create that string on 637

Reply:
That string is for the readable_model output